resources/Spelunky2RoomCodes.json

```diff
@@ -159,7 +159,7 @@
     "67": { "name": "Horizontal and vertical double room", "color": "gray" },
     "68": { "name": "Volcana trash room", "color": "gray" },
     "69": { "name": "Horizontal and vertical double room", "color": "gray" },
-    "6a": { "name": "Horizontal and vertical double room", "color": "gray" },
+    "6a": { "name": "Dead are Restless Bigroom", "color": "gray" },
```

Comment:
They is a lot to change in this file, I'm still thinking if we should use those names or maybe the enum names from https://github.com/spelunky-fyi/overlunky/blob/main/docs/game_data/room_templates.txt

Reply:
I'd like to just leave this one for now to at least improve this here.

My opinion is that the enum names aren't very reader friendly in all cases, so our names if fixed to be more accurate I prefer.

Reply:
A way to toggle them or to see both would be really cool